### PR TITLE
Quick search metrics

### PR DESCRIPTION
**Pull request by dippindots**
Fix https://github.com/cBioPortal/cbioportal/issues/5601.

Add a new quick-search report in google analytics:
image

### CODE REVIEW

File: src/shared/components/query/QueryAndDownloadTabs.tsx [hunk 1]
```diff
@@ -85,7 +91,7 @@ export default class QueryAndDownloadTabs extends React.Component<IQueryAndDownl
 					<MSKTab id={"advanced"} linkText={"Query"} onTabDidMount={()=>this.setDefaultTab(undefined)}>
                         <QueryContainer onSubmit={this.props.onSubmit} store={this.store}/>
 					</MSKTab>
-					<MSKTab id={QUICK_SEARCH_TAB_ID} linkText={<span>Quick Search <strong className={"beta-text"}>Beta!</strong></span>}  onTabDidMount={()=>this.setDefaultTab(QUICK_SEARCH_TAB_ID)}>
+					<MSKTab id={QUICK_SEARCH_TAB_ID} linkText={<span>Quick Search <strong className={"beta-text"}>Beta!</strong></span>}  onTabDidMount={()=>{this.setDefaultTab(QUICK_SEARCH_TAB_ID), this.trackQuickSearch()}}>
```
[the remark this answers is not in the document]
Comment (dippindots) [reply src/shared/components/query/QueryAndDownloadTabs.tsx]: Thanks! updated the solution, also updated a custom Google analytics report setting in the description.

File: src/shared/components/query/QueryAndDownloadTabs.tsx [hunk 2]
```diff
@@ -85,7 +91,7 @@ export default class QueryAndDownloadTabs extends React.Component<IQueryAndDownl
 					<MSKTab id={"advanced"} linkText={"Query"} onTabDidMount={()=>this.setDefaultTab(undefined)}>
                         <QueryContainer onSubmit={this.props.onSubmit} store={this.store}/>
 					</MSKTab>
-					<MSKTab id={QUICK_SEARCH_TAB_ID} linkText={<span>Quick Search <strong className={"beta-text"}>Beta!</strong></span>} hide={!this.props.showQuickSearchTab} onTabDidMount={()=>this.setDefaultTab(QUICK_SEARCH_TAB_ID)}>
+					<MSKTab id={QUICK_SEARCH_TAB_ID} linkText={<span>Quick Search <strong className={"beta-text"}>Beta!</strong></span>} hide={!this.props.showQuickSearchTab} onTabDidMount={()=>{this.setDefaultTab(QUICK_SEARCH_TAB_ID); this.trackQuickSearch();}}>
```
Comment (inodb) [inline src/shared/components/query/QueryAndDownloadTabs.tsx]: @dippindots 	maybe split this element over multiple lines, kinda hard to read

Comment (inodb) [reply src/shared/components/query/QueryAndDownloadTabs.tsx]: this should prolly only be called when google analytics is enabled. There should be some property

Comment (alisman) [reply src/shared/components/query/QueryAndDownloadTabs.tsx]: @inodb this is actually handled upstream.  the trackEvent function will basically just noop if there is no instance of ga.   i think this is a lot cleaner than having to check everywhere we want to track

**Pull request merged**